ledger-core/v2/cmd/protoc-gen-ins/plugins/extra/polymorph.go

```diff
@@ -30,6 +30,9 @@ func (p *Polymorph) GenerateMsg(file *generator.FileDescriptor, message *generat
 		return
 	}
 	id := insproto.GetPolymorphID(message.DescriptorProto)
+	if id == 0 {
```

Comment:
the previous 
if !insproto.HasPolymorphID(message.DescriptorProto) 
is not needed then

Reply:
Fixed